server/datastore/mysql/apple_mdm.go

```diff
@@ -653,6 +653,11 @@ func updateMDMAppleHostDB(
 		return ctxerr.Wrap(ctx, err, "update mdm apple host")
 	}
 
+	// clear any host_mdm_actions following re-enrollment here
```

Comment:
Just to double-check my understanding: `IngestMDMAppleDeviceFromCheckin` (from where this gets called) only gets called once for a device, when it enrolls via macOS MDM, right?

Reply:
for our future reference, this is the bug: https://github.com/fleetdm/fleet/issues/17232

Reply:
:+1: I'll update the ticket to cover the `host_mdm_actions` case too - i.e. so that a cert renewal doesn't end up removing a lock/unlock/wipe info. Since I think the fix will need to disambiguate between initial enrollment and cert renewal Authenticate, it will probably be trivial to fix both issues (profiles and mdm actions) once we can differentiate those calls.

Reply:
thank you! yes, you're right, sorry for missing that.